packages/forms-web-app/tests/unit/validators/custom/mime-binary-type.test.js

```diff
@@ -0,0 +1,86 @@
+const CrossBlob = require('cross-blob');
+const {
+  MIME_BINARY_TYPE_DOC,
+  MIME_TYPE_DOCX,
+  MIME_TYPE_PDF,
+  MIME_TYPE_JPEG,
+  MIME_TYPE_TIF,
+  MIME_TYPE_PNG,
+} = require('../../../../src/lib/mime-types');
+
+const validateMimeBinaryType = require('../../../../src/validators/custom/mime-binary-type');
+
+function MockFile() {}
+
+MockFile.prototype.create = function (name, size, mimeType, filePath) {
+  function range(count) {
+    let output = '';
+    for (let i = 0; i < count; i += 1) {
+      output += 'a';
+    }
+    return output;
+  }
```

Comment:
it might be possible to replace this entirely with `repeat` - https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat

`const blobStream = new CrossBlob(['a'.repeat(size)], { type: mimeType });`

Reply:
Changed.